API/src/TempusHub.API/Features/Demos/Services/StvParser.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Diagnostics;
+using System.Text.Json;
+
+namespace TempusHub.API.Features.Demos.Services;
+
+public static class StvParser
+{
+    public static StvParserResponse ExtractStvData(string fileName)
+    {
+        var processStartInfo = new ProcessStartInfo
+        {
+            FileName = "parse_demo.exe",
```

Comment:
move to config. locally this will be an exe, hosted this will probably be linux, not an EXE and different dir maybe.